cli/cmd/install_addon_test.go

```diff
@@ -20,11 +21,19 @@ func TestAddOnRender(t *testing.T) {
 	withTracingAddonValues.Tracing["enabled"] = true
 	addFakeTLSSecrets(withTracingAddonValues)
 
+	withExistingGrafana, err := testInstallOptions()
+	if err != nil {
+		t.Fatalf("Unexpected error: %v\n", err)
```

Comment:
I think you can format errors as strings, right?

Reply:
I see that its a common pattern in the tests to use %v instead of strings!